src/core/cli_arg.h

```diff
@@ -29,6 +31,7 @@ class CliArg {
     std::string _path{};
     int _port{0};
     int _baudrate{0};
+    bool _flow_control{false};
```

Comment:
```suggestion
    bool _flow_control_enabled{false};
```
or `_flow_control_enabled` so that it's clear what `true` means.